include/erlcloud_aws.hrl

```diff
@@ -42,7 +42,7 @@
           secret_access_key::string()|undefined|false,
           security_token=undefined::string()|undefined,
           %% epoch seconds when temporary credentials will expire
-          expiration :: pos_integer(),
+          expiration=undefined :: pos_integer()|undefined,
```

Comment:
I have no strong feeling on this. As long as the specs correctly say it can be undefined, knowing that this is the default value for records where the field is not set seems fine. Explicitly calling out the default is helpful but doesn't seem needed to me.